dotnet/src/SemanticKernel.Core/PromptTemplate/KernelPromptTemplate.cs

```diff
@@ -163,5 +181,16 @@ void AddIfMissing(string variableName)
             }
         }
     }
+
+    private static bool ShouldEncodeTags(bool disableTagEncoding, List<string> safeBlocks, Block block)
+    {
+        if (block is VarBlock varBlock)
+        {
+            return !safeBlocks.Contains(varBlock.Name);
+        }
+
+        return !disableTagEncoding && block is not TextBlock;
```

Comment:
Should it be `block is TextBlock`?  I would have thought we should encode only if it's a TextBlock.

Reply:
A `TextBlock` is static text, so no need to encode this